bolt-simple-taproot.md

```diff
@@ -764,43 +764,50 @@ any keys aggregated via musig2 also are assumed to use the `KeySort` routine
 #### To Local Outputs
 
 For the simple taproot commitment type, we use the same flow of revocation or
-delay, but instead left the revocation case into the taproot key-spend case.
+delay, while re-using the NUMS point to ensure that the internal key is always
+revealed. This ensures that the anchor outputs can _always_ be spent given
+chain revelead information.
 
 The new output has the following form:
 
   * `OP_1 to_local_output_key`
   * where:
-    * `to_local_output_key = revocationpubkey + tagged_hash("TapTweak", revocationpubkey || to_delay_script_root)`
-    * `to_delay_script_root = tapscript_root([to_delay_script])`
+    * `to_local_output_key = revocationpubkey + tagged_hash("TapTweak", taproot_nums_point || to_delay_script_root)`
```

Comment:
s/revocationpubkey/taproot_nums_point

Reply:
I think the other way around. We need to keep the `revocationpubkey` for the to_local output. The to_remote is where we want the NUMS point to be able to scan for remotely closed channels on chain.

Reply:
Yep that was an oversight, this is just the long form of the tapscript top-level tweak. 

We want the NUMs key here, but then we'll reveal the to_local key in an extra data push.

Reply:
> I dont agree. We cant have the to_local_delay be the internal key here because then it is spendable by the local part immediately via the key spend path

Ah of course, you're totally right. I failed to reload all context before reviewing this PR and the scripts one in lnd. Sorry for the confusion!